src/__tests__/index.js

```diff
@@ -372,6 +372,107 @@ test('should receive inner ref if specified', () => {
   expect(getRef).toHaveBeenCalled()
 })
 
+test('should pass inner ref to underlying component if forwarded', () => {
+  let nodeType = null
```

Comment:
Instead of `nodeType`, let's call this `node` :+1:
Same with other similar tests.

Reply:
done!